IFC4x3/Properties/i/Increasing_34FH6g00XEcAJrdjKZ3gOO/Documentation.md

```diff
@@ -0,0 +1,5 @@
+It is meant for use in referents of predefined type REFERENCEMARKER being nested to a given alignment.
```

Comment:
I would propose to add somewhere that these are not supposed to be edited by an end user as a sort of a informal agreement. What do you think?

Reply:
Here are some thoughts, maybe you can read an answer or a viewpoint from the following paragraphs:

As I see its use, the REFERENCEMARKER referents would have been created by one authoring tool and then exported to IFC format. An importing tool would then import the IFC data. It is our our task is to convey to all parties how these things are meant to be interpreted.

One possible use case is a long-term database storage for alignment data, which will be a repository for all users in the years to come. Many different tools will read the data, none of them would change data upon import. Another use case is an IFC Viewer which will be able to compute new STATION measurements based on the REFERNCEMARKER referents and the associated Pset(s) found in the imported IFC file. 

==> None of the above need to change data.

The other type of use case is a brown-field modification project, where everything is subject to modification. This would require a tool which builds an internal model from the imported IFC data, then allows the user to make changes, then exports to IFC again.

Reply:
Would this final scenario not rely on the tool to make the judgement on the value of the property or should it be manually modified by an end user?

Reply:
Please let us stash this question until our upcoming meeting on Reference Alignment System , as I don't think I get the problem here :-(